apps/examples/security_test/seclink/sl_auth_test.c

```diff
@@ -86,7 +86,7 @@ static hal_data g_rand = HAL_DATA_INITIALIZER;
 static hal_data g_cert_in = HAL_DATA_INITIALIZER;
 static hal_data g_cert_out = HAL_DATA_INITIALIZER;
 
-ST_SET_PACK(sl_auth);
+ST_SET_PACK1(sl_auth);
```

Comment:
how about to rename the macro. for example ST_SET_PACK_GLOBAL? It doesn't seem meaningful to add 1

Reply:
done